src/tests/ftest/osa/offline_extend.py

```diff
@@ -95,6 +97,17 @@ def run_offline_extend_test(self, num_pool, data=False, oclass=None):
             if self.test_during_aggregation is True and (num_pool > 1):
                 self.delete_extra_container(self.pool)
             output = self.pool.extend(rank_val)
+            self.log.info(output)
+            if exclude_or_drain == "exclude":
+                self.pool.wait_for_rebuild_to_start()
+                sleep(4)
```

Comment:
You might want to add comment describing why we need 4 sec sleep here.

Reply:
done.